auth0/README.md

```diff
@@ -57,6 +57,5 @@ Need help? Contact [Datadog support][1].
 
 [1]: https://docs.datadoghq.com/help
 [2]: https://manage.auth0.com
-[3]: https://auth0.com/docs/logs/streams/datadog
```

Comment:
Also need to remove the reference on line 9.

Reply:
removed reference to link from line 9 👍